### PR TITLE
[12.0][FIX] hr_timesheet_sheet: consistency with v11

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@astirpe @mreficent @pedrobaeza @victormmtorres please decide if this one is needed

File: hr_timesheet_sheet/tests/test_hr_timesheet_sheet.py [hunk 1]
```diff
@@ -399,11 +399,12 @@ def test_6(self):
         self.assertEqual(len(timesheet_3_4_and_5), 3)
 
         timesheet_6 = self.aal_model.create({
-            'name': empty_name,
+            'name': 'z',
```
Comment (astirpe) [inline hr_timesheet_sheet/tests/test_hr_timesheet_sheet.py:402]: Why you change this?

**Review by MiquelRForgeFlow — APPROVED** [review]
@astirpe 

As I was who originally created this tests, here in this timesheet_6 I wanted to test the behavior of creating a timesheet with name and then, test the behavior of changing the name to an empty_name.

If you create here the timesheet_6 directly with an empty_name, then you are not testing the behavior I explained above.

File: hr_timesheet_sheet/tests/test_hr_timesheet_sheet.py (hunk 1 above)
Comment (astirpe) [reply hr_timesheet_sheet/tests/test_hr_timesheet_sheet.py:402]: Question answered. Thanks.

**Review by astirpe — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@mreficent then the test has changed its intention?

**Comment by MiquelRForgeFlow** [comment]
Yeah, and this PR is restoring the initial intention.

**Pull request merged**